django/db/models/fields/related.py

```diff
@@ -1741,6 +1778,12 @@ def contribute_to_related_class(self, cls, related):
 
 
 class ForeignKey(ForeignObject):
+    # Field flags
+    one_to_many = True
```

Comment:
umm.... am I crazy? Aren't ForeignKeys many_to_one? and the reverse of a ForeignKey is one_to_many?

Right? if Book has ForeignKey(Author), then many Books have one Author and one Author has many Books?